altapay.php

```diff
@@ -31,24 +31,24 @@ class ALTAPAY extends PaymentModule
 
     public function __construct()
     {
-        $this->name                   = 'altapay';
-        $this->tab                    = 'payments_gateways';
-        $this->version                = '3.3.1';
-        $this->v16                    = _PS_VERSION_ >= '1.6.1.24';
-        $this->v17                    = _PS_VERSION_ >= '1.7.7.0';
-        $this->author                 = 'AltaPay A/S';
-        $this->is_eu_compatible       = 1;
+        $this->name = 'altapay';
+        $this->tab = 'payments_gateways';
+        $this->version = '3.3.1';
+        $this->v16 = _PS_VERSION_ >= '1.6.1.24';
```

Comment:
This is not a safe way to compare version:
1.6.1.24 would bee is seen as older than 1.6.1.3.
Instead, we should use `version_compare`